habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hab3_hl_neural.yaml

```diff
@@ -7,17 +7,25 @@ hierarchical_policy:
   high_level_policy:
     name: "NeuralHighLevelPolicy"
     allowed_actions:
-      - nav
+      - nav_to_goal
+      - nav_to_obj
       - pick
       - place
-      - nav_to_receptacle_by_name
```

Comment:
Shall we uncomment this?

Reply:
Hmm this is just for the single agent policy, so I think no need to introduce it.